admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/entity-type-delete-confirm.jsp

```diff
@@ -1,33 +1,53 @@
 <%@ taglib prefix="s" uri="/struts-tags" %>
 <%@ taglib prefix="wpsf" uri="/apsadmin-form" %>
 
-<h1 class="panel panel-default title-page">
-	<span class="panel-body display-block">
-		<a href="<s:url action="initViewEntityTypes" namespace="/do/Entity"><s:param name="entityManagerName"><s:property value="entityManagerName" /></s:param></s:url>" title="<s:text name="note.goToSomewhere" />: <s:text name="title.entityAdmin.manager" />&#32;<s:property value="entityManagerName" />">
-		<s:text name="%{'title.' + entityManagerName + '.management'}" /></a>
-		&#32;/&#32;
+<ol class="breadcrumb page-tabs-header breadcrumb-position">
+	<li>
+		<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+	</li>
+	<li class="page-title-container">
 		<s:text name="title.entityTypes.editType.remove" />
-	</span>
+	</li>
+</ol>
+<h1 class="page-title-container">
+	<s:text name="title.entityTypes.editType.remove"/>&#32;<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+	<span class="pull-right">
+        <a tabindex="0" role="button" data-toggle="popover" data-trigger="focus" data-html="true" title=""
+		   data-content="TO be inserted" data-placement="left" data-original-title=""><i class="fa fa-question-circle-o"
+																						 aria-hidden="true"></i></a>
+    </span>
 </h1>
+<div class="text-right">
+	<div class="form-group-separator"></div>
+</div>
+<br>
 
-<s:form action="removeEntityType">
-	<p class="sr-only">
-		<wpsf:hidden name="entityManagerName" />
-		<wpsf:hidden name="entityTypeCode" />
-	</p>
-	<div class="alert alert-warning">
+
+<div class="text-center">
+	<s:form action="removeEntityType">
+		<p class="sr-only">
+			<wpsf:hidden name="entityManagerName" />
+			<wpsf:hidden name="entityTypeCode" />
+		</p>
+		<i class="fa fa-exclamation esclamation-big" aria-hidden="true"></i>
+		<p class="esclamation-underline">
+			<s:text name="title.entityTypes.editType.remove"/>&#32;<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+		</p>
 		<p>
```

Comment:
Tenete sempre questa struttura quando fate una pagina di cancellazione. Togliete i tag <code> in questo caso. 
 
`<p class="esclamation-underline-text"><s:property value="%{getPage(nodeToBeDelete).getDraftTitle(currentLang.getCode())}" />&nbsp;?</p>`